examples/hybrid/sphere/deformation_flow.jl

```diff
@@ -221,12 +221,15 @@ function run_deformation_flow(use_limiter, fct_op)
         zd = z - z_c
 
         centers = (
-            Geometry.LatLongZPoint(ϕ_c, λ_c1, FT(0)),
-            Geometry.LatLongZPoint(ϕ_c, λ_c2, FT(0)),
+            Geometry.LatLongZPoint(ϕ_c, λ_c1, z),
+            Geometry.LatLongZPoint(ϕ_c, λ_c2, z),
```

Comment:
Was this a bug fix? Can you explain this change?

Reply:
@charleskawczynski This follows updates to the `GlobalSphericalGeometry` : see the great circle function [here](https://github.com/CliMA/ClimaCore.jl/blob/c81d8573b0e8b28a552548bc2d723bb5b997f59f/src/Geometry/globalgeometry.jl#L170-L199) in this PR, it relaxes the shallow-sphere assumption